### DISCUSSION TITLE
Powershell call redirect

### DISCUSSION THREAD

**Discussion by Marvin700** [community/community#38676] [Other Feature Feedback, Questions, & Ideas]
Hello, 
Is there a setting to Redirect Traffic from a Powershell command ( irm GitHub... | iex) to a specific File ?

So i Could use my repo Link, instead of repo/Release/File.. ?

**Comment by ossanna16** [comment]
Hi there @Marvin700 and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)

**Comment by Marvin700** [comment]
Done with Cloudflare :)
